### ISSUE TITLE
where is minicap.so for android sdk 34?

### ISSUE THREAD

**Issue by mchen12345** [DeviceFarmer/minicap#50]
**What is the issue or idea you have?**

**Have you tried STF?**

**Comment by onlinesen** [comment]
+1, any help will be appreciated!

**Comment by fanlb** [comment]
+1, any help will be appreciated!

**Comment by varundtsfi** [comment]
Complete AOSP is not yet available due to that we have to wait for that. Once that is coming we can try to build and provide you the minicap.so for android 14.

**Comment by varundtsfi** [comment]
Hi 
Now approach for getting the screenshots is getting change from  minicap.so to apk. DeviceFarmer Team is  using apk for that so thare is not minicap.so support in future too. 
@koral-- right?

Batter don't post any query here.

**Comment by koral--** [comment]
Yes, that's correct. See e.g. https://github.com/DeviceFarmer/minicap/issues/41#issuecomment-1422198153

**Comment by wangvic21** [comment]
Compiled for arm64-v8a. 

[minicap.so.zip](https://github.com/user-attachments/files/24998982/minicap.so.zip)
